### DISCUSSION TITLE
Sharing addon configuration between components

### DISCUSSION THREAD

**Discussion by strk** [WeblateOrg/weblate#9937] [Q&A]
We're using a multi-component setup for our project but would like to configure addons only once and see those configuration applied on all components and I could not figure out how to do so. Can you help ?

In particular we're experimenting with the "Customize gettext output" addon configuration (for line wrapping) and every time we have to test something it takes touching over 40 components which is clearly impractical.

**Comment by nijel** [comment]
There is no direct support for this, but you can utilize API to sync these externally, see https://docs.weblate.org/en/latest/api.html#add-ons

**Comment by strk** [comment]
I found a previous discussion about this in https://github.com/WeblateOrg/weblate/issues/8564 - would it make sense to file an enhancement ticket to make it easier to share configuration between components of the same project ?

**Reply by nijel** [reply]
I think it would make sense to install add-ons at the project level as well. This might overlap with https://github.com/WeblateOrg/weblate/issues/7604, though.

**Reply by nijel** [reply]
Created https://github.com/WeblateOrg/weblate/issues/10051 for this.
